### PR TITLE
Fix profile update bug

### LINKED ISSUE #265: Bug updating profile
I was testing the instructions in the README as part of #259 and noticed two issues
* [This line](https://github.com/marcua/ayb/blob/49a0c21e59cfa6f8bc29cfcebb66037498710c4c/README.md?plain=1#L109) references `--link` when it should reference `--links`
* When I run the example command, I get an error
```bash
$ cargo run -- client update_profile marcua --display_name 'Adam Marcus' --links 'http://marcua.net'
    Finished dev [unoptimized + debuginfo] target(s) in 0.28s
     Running `target/debug/ayb client update_profile marcua --display_name 'Adam Marcus' --links 'http://marcua.net'`
Error: Database(PgDatabaseError { severity: Error, code: "42601", message: "syntax error at or near \",\"", detail: None, hint: None, position: Some(Original(18)), where: None, schema: None, table: None, column: None, data_type: None, constraint: None, file: Some("scan.l"), line: Some(1145), routine: Some("scanner_yyerror") })
$ cargo run -- client profile marcua
    Finished dev [unoptimized + debuginfo] target(s) in 0.27s
     Running `target/debug/ayb client profile marcua`
 Display name | Description | Organization | Location | Links 
--------------+-------------+--------------+----------+-------
 null         | null        | null         | null     |  
$ cargo run -- client update_profile marcua --display_name 'Adam Marcus' 
    Finished dev [unoptimized + debuginfo] target(s) in 0.29s
     Running `target/debug/ayb client update_profile marcua --display_name 'Adam Marcus'`
Error: Database(PgDatabaseError { severity: Error, code: "42601", message: "syntax error at or near \",\"", detail: None, hint: None, position: Some(Original(18)), where: None, schema: None, table: None, column: None, data_type: None, constraint: None, file: Some("scan.l"), line: Some(1145), routine: Some("scanner_yyerror") })

```

### PR DESCRIPTION
**Pull request by marcua**
Resolves #265

### CODE REVIEW

**Review by sofiaritz — CHANGES_REQUESTED** [review]
Exactly what I had in mind :)
Could you add a test case in the E2E tests to catch this bug in the future if there's some regression?

File: tests/e2e.rs [hunk 1]
```diff
@@ -586,6 +586,26 @@ fn client_server_integration(
     )?;
 
     // Make some partial profile updates and verify profile details upon retrieval
+    update_profile(
```
Comment (marcua) [inline tests/e2e.rs:589]: Fails on `main`, succeeds in this branch

**Comment by marcua** [comment]
Great idea, added! :)

**Comment by marcua** [comment]
I'll merge this, but if you spot any other issues, I'm happy to iterate in another PR! :)

**Pull request merged**